lib/beaker/subcommands.rb

```diff
@@ -0,0 +1,20 @@
+require "thor"
+require "fileutils"
+require "beaker/subcommand/subcommand_util"
+
+module Beaker
+  class Subcommands < Thor
```

Comment:
@ferglor it makes more sense to me that the module is called `SubCommands` and the class called `SubCommand`. Does that make sense? The pluralization makes more sense for the module name to me...

@kevpl what do you think?